config/symbols.us.txt

```diff
@@ -636,59 +639,16 @@ background_objects_2_4C = 0x80141aa4;
 background_objects_2_4D = 0x80141aa5;
 background_objects_2_4E = 0x80141aa6;
 
-
-engine_obj = 0x801721C0;
-engine_obj_01 = 0x801721C1;
-engine_obj_02 = 0x801721C2;
-engine_obj_03 = 0x801721C3;
+engine_obj = 0x801721C0; // size:0x64
```

Comment:
There's a tradeoff here in that if I use D_801721CF (engine_obj_0F) for example, it doesn't produce a compile error, while the current method does. Is the motivation getting better scratches with the automatic `+ offset`?

I suppose it's more important to do this step to fix existing decompiled functions with fake variables, since all new ones should be obvious that they are part of engine_obj

Reply:
Ah I didn't think about that. Most of the time I use `engine_obj.unkF` and just let the scratch not match (or override the match), then verify it matches when imported into the full build.

I'll revert the deletions.

Reply:
Ah you're saying that `D_801721CF` will show up in the undefined symbols. I think that's due to mixing `// size:0x64` with overlapping named symbols. If I delete `engine_obj_stage` etc it correctly errors out on `D_801721CF`.

Reply:
Downside of that approach is that as more fields are named, they'll need to be hand-updated on import. it's easier to replace `engine_obj_stage` -> `engine_obj.stage` than `engine_obj.unkC` -> `engine_obj.stage`.

Reply:
I'm not able to test right now but from what I remember, we for example had references to D_801721CF in the codebase. When I added engine_obj_0F, D_801721CF became a linker error because splat didn't define it. This helped with cleaning that stuff up. If you use the `size`, using the wrong symbol works right?